game/client_types/logic.js

```diff
@@ -110,7 +110,7 @@ module.exports = function(treatmentName, settings, stager, setup, gameRoom) {
                 lastDumpSize = s;
                 node.game.memory.save('.db.json', function() {
                     fs.createReadStream(gameRoom.dataDir + '.db.json')
-                        .pipe(fs.createWriteStream(gameRoom.dataDir + 'db.json'));
+                        .pipe(fs.createWriteStream(gameRoom.dataDir + '.db.json'));
```

Comment:
This is not correct, still does not save inside the room folder. Use the path module to join paths.

```
path.join(gameRoom.dataDir,  'db.json')
```